sdk/storage/azure-storage-blobs/swagger/README.md

```diff
@@ -609,6 +587,27 @@ directive:
       $.get.responses["200"].schema["$ref"] = "#/definitions/BlobServiceProperties";
 ```
 
+### SetBlobServiceProperties
```

Comment:
Can we add an item under section `### Define names for return types` instead of creating a new section? Does these two methods have the same effect?

Reply:
I've tested for this. It has the same effect. I will change these scenarios.